worldedit-core/src/main/java/com/sk89q/worldedit/command/NavigationCommands.java

```diff
@@ -75,6 +77,8 @@ public void unstuck(Player player) throws WorldEditException {
     public void ascend(Player player,
                        @Arg(desc = "# of levels to ascend", def = "1")
                            int levels) throws WorldEditException {
+        World world = player.getWorld();
+        checkArgument(levels >= 1 && levels <= (world.getMaxY() - world.getMinY()), "1 <= levels <= world height");
```

Comment:
```suggestion
        checkArgument(1 <= levels && levels <= (world.getMaxY() - world.getMinY()), "1 <= levels <= world height");
```
This makes it look more like the error message, and I prefer to have bounds done this way.